pkg/controller/masteruserrecord/masteruserrecord_controller.go

```diff
@@ -95,6 +82,16 @@ func (r *ReconcileMasterUserRecord) Reconcile(request reconcile.Request) (reconc
 		return reconcile.Result{}, err
 	}
 
-	// Do nothing for now
+	for _, account := range userRecord.Spec.UserAccounts {
+		syncStatus := syncWithUserAccount(account, userRecord)
+		if syncStatus.Requeue {
+			return reconcile.Result{
+				Requeue:      syncStatus.Requeue,
+				RequeueAfter: syncStatus.RequeueAfter},
+				syncStatus.Err
+		}
+		continue
+	}
```

Comment:
We will also need to update the status of the MasterUserRecord but I guess this is in the TODO list :)
Just don't forget that we will need to update the status if sync failed too.

Reply:
yes, there are much more things that are missing in the `masteruserrecord_controller.go` code